migrations/0000_loresmith_db_schema.sql

```diff
@@ -1,7 +1,7 @@
 -- Initial schema for LoreSmith campaigns and RAG functionality
 
 -- Campaigns table (replacing KV storage)
-CREATE TABLE campaigns (
+CREATE TABLE IF NOT EXISTS campaigns (
   id TEXT PRIMARY KEY,
```

Comment:
like. can we lowercase all plz? no sarcasm

Reply:
the SQL keywords? sure